From 01fec8c0f6a4a87186979f510b7aa77b2931ed6b Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Tue, 8 Feb 2022 07:25:15 -0700
Subject: [PATCH] Update generic.py

---
 pyiron_base/project/generic.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyiron_base/project/generic.py b/pyiron_base/project/generic.py
index fab57e1d..9c42afef 100644
--- a/pyiron_base/project/generic.py
+++ b/pyiron_base/project/generic.py
@@ -12,6 +12,7 @@
 import pandas
 import pint
 import importlib
+import math
 import numpy as np
 
 from pyiron_base.project.maintenance import Maintenance
