From 81e11202cc8be888e04065e83c269cfbe88c8812 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan.janssen@outlook.com>
Date: Thu, 21 Jul 2022 22:39:51 -0600
Subject: [PATCH] Fix import loop

---
 pyiron_base/generic/filedata.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pyiron_base/generic/filedata.py b/pyiron_base/generic/filedata.py
index 8700c9282..22966d394 100644
--- a/pyiron_base/generic/filedata.py
+++ b/pyiron_base/generic/filedata.py
@@ -10,7 +10,8 @@
 
 import pandas
 
-from pyiron_base import ImportAlarm, FileHDFio, ProjectHDFio
+from pyiron_base.generic.hdfio import FileHDFio, ProjectHDFio
+from pyiron_base.generic.util import ImportAlarm
 
 __author__ = "Niklas Siemer"
 __copyright__ = (
