From daecdd3ffd7caf2624a1bf5ddf4c60d5ee2102fd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Jan=C3=9Fen?= <janssen@mpie.de>
Date: Tue, 28 Apr 2026 22:29:33 +0200
Subject: [PATCH] fix h5py version

---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index eaaad52..9360afe 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -26,7 +26,7 @@ dependencies = [
     "cloudpickle>=2.2.1,<=3.1.2",
     "executorlib>=0.1.0,<=1.9.2",
     "h5io_browser>=0.0.14,<=0.2.14",
-    "h5py>=3.11.0,<=3.15.1",
+    "h5py>=3.11.0,<=3.16.0",
     "numpy>=1.26.0,<=2.4.3",
     "monty>=2025.1.3,<=2026.2.18",
     "pandas>=2.0.0,<=3.0.2",
