From f160acf0ef41fd414e0c69d6a360def95fb1680b Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan.janssen@outlook.com>
Date: Thu, 21 Jul 2022 22:25:12 -0600
Subject: [PATCH] Add file data export

---
 pyiron_base/__init__.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pyiron_base/__init__.py b/pyiron_base/__init__.py
index aa5909bf1..fb10dd41b 100644
--- a/pyiron_base/__init__.py
+++ b/pyiron_base/__init__.py
@@ -11,6 +11,7 @@
 from pyiron_base.generic.datacontainer import DataContainer
 from pyiron_base.generic.inputlist import InputList
 from pyiron_base.generic.parameters import GenericParameters
+from pyiron_base.generic.filedata import load_file, FileDataTemplate, FileData
 from pyiron_base.generic.util import Deprecator, deprecate, deprecate_soon, ImportAlarm
 from pyiron_base.job.executable import Executable
 from pyiron_base.job.external import Notebook, load, dump
