From 9dec1a4205ba9cac37bcf608606ba8692f0e9fa4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Jan=20Jan=C3=9Fen?= <janssen@mpie.de>
Date: Fri, 6 Jun 2025 11:43:07 +0200
Subject: [PATCH] Add add_module_conversion to __init__.py

---
 pyiron_base/__init__.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/pyiron_base/__init__.py b/pyiron_base/__init__.py
index de39789bb..33b5844c1 100644
--- a/pyiron_base/__init__.py
+++ b/pyiron_base/__init__.py
@@ -29,7 +29,7 @@
 from pyiron_base.jobs.master.interactivewrapper import InteractiveWrapper
 from pyiron_base.jobs.master.list import ListMaster
 from pyiron_base.jobs.master.parallel import JobGenerator, ParallelMaster
-from pyiron_base.maintenance.generic import LocalMaintenance, Maintenance
+from pyiron_base.maintenance.generic import add_module_conversion, LocalMaintenance, Maintenance
 from pyiron_base.project.decorator import job
 from pyiron_base.project.external import Notebook, dump, load
 from pyiron_base.project.generic import Creator, Project
@@ -106,6 +106,7 @@
     HasGroups,
     HasHDF,
     Toolkit,
+    add_module_conversion,
     LocalMaintenance,
     Maintenance,
 ]
